scipy-1.0/covlett.tex

```diff
@@ -5,70 +5,102 @@
 
 % to adjust vertical spacing properties of
 % newlfm class see: https://tex.stackexchange.com/a/65533/43006
-\topmarginsize{2mm}
+%\unprtop{0mm}
+\topmarginsize{0mm}
 \topmarginskip{0in}
 \headermarginsize{0in}
 \headermarginskip{0in}
+\MinHead{0in}
+%\dateskipafter{0pt}
+%\dateskipbefore{0pt}
+\sigskipbefore{40pt}
+%\sigskipafter{0pt}
+\closeskipbefore{1pt}
+%\closeskipafter{0pt}
+%\addrfromskipbefore{0pt}
+\addrfromskipafter{0in}
+\addrtoskipbefore{0in}
+%\addrtoskipafter{0pt}
+\newlfmP{Headlinewd=0pt,Footlinewd=0pt}
+\newlfmP{sigsize=10pt}
+\bottommarginskip{60pt}
+\MinFoot{0pt}
 
 \usepackage{kpfonts}
+\usepackage{hyperref}
 \usepackage{url}
 
+\hypersetup{%
+  linkbordercolor=blue,% hyperlink borders will be blue
+  pdfborderstyle={/S/U/W 1}% border style will be underline of width 1pt
+}
 \widowpenalty=1000
 \clubpenalty=1000
 
-\newlfmP{headermarginskip=10pt}
-\newlfmP{sigsize=30pt}
-\newlfmP{dateskipafter=20pt}
-%\newlfmP{addrfromphone}
-%\newlfmP{addrfromemail}
-%\PhrPhone{Phone}
-%\PhrEmail{Email}
-
 \namefrom{Ralf Gommers,
           Matt Haberland,
           and Tyler Reddy}
 \addrfrom{%
     \today\\[10pt]
+    Ralf Gommers,\\
     Quansight Labs,\\
-    The Netherlands\\
+    The Netherlands,\\
```

Comment:
It looks like the commas were added intentionally, but I've never seen that before. Is it specified by the journal or something? [Example cover letters](https://www.google.com/search?rlz=1C1GCEU_enUS820US820&biw=1920&bih=937&tbm=isch&sa=1&ei=fWs7XcjCIZfN-gSpxKbwBg&q=academic+cover+letter+format&oq=academicover+letter+format&gs_l=img.3.0.0i7i30.8188.9275..10149...0.0..0.90.509.7......0....1..gws-wiz-img.......0i8i7i30.YEzf8ENXY08#imgrc=GL-Dc68AJu8TIM:).

Reply:
I'm happy to surrender in this battle :)

Reply:
Was there a battle before? Or you have an argument for the comas?

Reply:
Just a habit of using commas on all but the last line of a contact or address block

Reply:
Ok, I've purged the commas